landoapi/hg.py

```diff
@@ -556,19 +556,26 @@ def push(self, target, bookmark=None):
             and self.patch_header("Fail HG Import") == b"LOSE_PUSH_RACE"
         ):
             raise LostPushRace()
+
+        extra_args = ["-f"] if force_push else []
```

Comment:
nit: This would be a little more future proof if `extra_args` was initialized as an empty list, and appended to as needed.